bin/handler-pagerduty.rb

```diff
@@ -49,6 +49,9 @@ def api_key
     @api_key ||=
       if @event['check']['pager_team']
         settings[json_config][@event['check']['pager_team']]['api_key']
+      elsif settings[json_config][@event['client']['pager_team']].nil?
```

Comment:
I think this should be:
```
elsif @event['client']['pager_team'] && settings[json_config][@event['client']['pager_team']].nil?
```

as we only want to fallback if we define it in the event but it is not found.